placeholder_accessors/index.md

```diff
@@ -192,6 +193,51 @@ The handler gains a new method,
 that registers the requirements of the placeholder accessor on the given
 command group.
 
+### Placeholder `accessor` without a buffer
+
+Not providing either the handler or the memory object
+makes the accessor default constructible,
+creating a "null" accessor.
+A null accessor can be bound to a buffer
+by constructing a new placeholder accessor with the buffer
+and assigning it to the null accessor.
+Example:
+
+```cpp
+accessor<T, dim, mode, target, access::placeholer::true_t> acc0;
+
+acc0 = accessor<T, dim, mode, target, access::placeholer::true_t>(buf);
+```
+
+To simplify placeholder accessor construction,
+a new free function `cl::sycl::codeplay::make_placeholder_accessor` is provided,
+which takes a buffer as an argument:
+
+```cpp
+template <access::mode mode, access::target target, typename T, int dim,
+          typename AllocatorT>
+accessor<T, dim, mode, target, access::placeholder::true_t>
+make_placeholder_accessor(buffer<T, dim, AllocatorT>& bufferRef);
+```
+
+A null accessor does not need to be registered with the command group,
+i.e. there is no need to call `require()` on it.
+It is allowed to pass a null accessor to a kernel,
+as long as it's not dereferenced inside the kernel,
+otherwise an out-of-bounds access will occur,
+likely leading to a segmentation fault or similar.
+If the null accessor has been bound to a buffer,
+it is no longer a null accessor,
+which means it has to be registered with the command group
+and can be dereferenced inside the kernel.
```

Comment:
This is also a bit confusing. A null accessor bound to a buffer, is then bound to that buffer forever? can it re-assigned? can it be assigned back to null? What happens if two command groups submit a different require bounding the same null accessor to two separate buffers?

Reply:
The main point of this MR is that the `require` function doesn't do the binding - it doesn't modify the accessor at all. So a null accessor can be "bound" by assigning to it a placeholder that already has an associated buffer.